From b105baf9b02c63bc9156d30cbbd731a6641f29c6 Mon Sep 17 00:00:00 2001
Message-ID: <b105baf9b02c63bc9156d30cbbd731a6641f29c6.1719691458.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1719691458.git.sharlatanus@gmail.com>
References: <cover.1719691458.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 18 Jun 2024 17:52:46 +0100
Subject: [PATCH 14/26] gnu: Add go-github-com-otiai10-mint.

* gnu/packages/golang-check.scm (go-github-com-otiai10-mint): New variable.

Change-Id: I2b203256756bf9dbeb838e2b289f4ffb9c902404
---
 gnu/packages/golang-check.scm | 25 +++++++++++++++++++++++++
 1 file changed, 25 insertions(+)

diff --git a/gnu/packages/golang-check.scm b/gnu/packages/golang-check.scm
index 00ea185cbc..be282ee3e9 100644
--- a/gnu/packages/golang-check.scm
+++ b/gnu/packages/golang-check.scm
@@ -697,6 +697,31 @@ (define-public go-github-com-onsi-gomega
 framework.")
     (license license:expat)))
 
+(define-public go-github-com-otiai10-mint
+  (package
+    (name "go-github-com-otiai10-mint")
+    (version "1.6.3")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/otiai10/mint")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "0g5zhz4znp68427p2a1yvrxbq90y7caagdd7zsb4iygnhdszfm7w"))))
+    (build-system go-build-system)
+    (arguments
+     (list
+      #:go go-1.21
+      #:import-path "github.com/otiai10/mint"))
+    (home-page "https://github.com/otiai10/mint")
+    (synopsis "Minimal assertion for Golang testing framework")
+    (description
+     "Mint (@code{mint.Mint}) is wrapper for @code{*testing.T} blending
+testing type to omit repeated @code{t}.")
+    (license license:expat)))
+
 (define-public go-github-com-pkg-profile
   (package
     (name "go-github-com-pkg-profile")
-- 
2.41.0

